src/robot_interface/robot_interface.py

```diff
@@ -45,12 +45,12 @@ def mission_scheduled(self) -> bool:
 
     @abstractmethod
     def mission_status(self, mission_id: Any) -> MissionStatus:
-        """Retrieves status of the current executing mission for the robot.
+        """Retrieves status of the current executing mission task for the robot.
 
         Parameters
         ----------
-        mission_id : Any
-            Unique identifier for the mission which the status should be checked for
+        task_id : UUID
+            Unique identifier for the task which the status should be checked for
```

Comment:
Docstring is renamed, but the parameter in the function def is still called mission_id